styles/globals.scss

```
@@ -35,6 +35,13 @@ body {
       flex-grow: 0;
     }
 
+    // override bootstrap so we don't wrap the hamburger
```

I understood this because of your screenshot but don't think I would have otherwise
```suggestion
    // override bootstrap so the hamburger menu stays next to EDD logo instead of overflowing to next line
```